packages/main/src/index.ts

```diff
@@ -8,11 +8,18 @@ import { PassThrough } from "stream"
 import * as cache from "./cache"
 import { contentType, filename } from "./models/MessagePart"
 import schema from "./schema"
+import * as path from "path"
 
 // TODO: We're having an issue checking the TLS certificate for Google's IMAP
 // service
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
 
+// TODO: Is there a better way to either load frontend assets from the asar, or
+// to reference unpacked assets?
+const appUrl = isDev
+  ? "http://localhost:3000"
+  : "file://" + path.join(__dirname, "..", "public", "index.html")
```

Comment:
Thanks to some Electron magic we can reference files that are actually packaged inside the asar as though they are ordinary files in the host filesystem.